### PR TITLE
Add `short` param for system_load segment

### PR DESCRIPTION
**Pull request by iblis17**

### CODE REVIEW

**Comment by ZyX-I** [comment]
Missing tests.

**Comment by iblis17** [comment]
@ZyX-I ok. I added the tests!

File: powerline/segments/common/sys.py [hunk 1]
```diff
@@ -13,7 +13,8 @@
 cpu_count = None
 
 
-def system_load(pl, format='{avg:.1f}', threshold_good=1, threshold_bad=2, track_cpu_count=False):
+def system_load(pl, format='{avg:.1f}', threshold_good=1, threshold_bad=2,
+		track_cpu_count=False, short=False):
```
Comment (ZyX-I) [inline powerline/segments/common/sys.py]: You should either leave everything on one line or align with first argument (note: *alignment* uses spaces).

Comment (iblis17) [inline powerline/segments/common/sys.py]: hmm... I just found that this code is using tab for indentation.
So, will using all spaces for this line be suitable? Or to keep this consistency in same file, should i leave it without new line?

**Comment by iblis17** [comment]
@ZyX-I ok. I got it.
I just figured it out from the `CONTRIBUTING.rst`.

**Pull request merged**